bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/ghost/Ghost.scala

```diff
@@ -294,11 +291,11 @@ object Ghost {
   val INSTRUMENT_NAME_PROP: String = "GHOST"
 
   // GHOST-specific exposure times.
-  val EXPOSURE_TIME_RED_PROP = "exposureTimeRed"
+  val EXPOSURE_TIME_RED_PROP = "redExposureTime"
   val DEF_EXPOSURE_TIME_RED = 10.0
   val EXPOSURE_TIME_RED_KEY = new ItemKey(INSTRUMENT_KEY, EXPOSURE_TIME_RED_PROP)
 
-  val EXPOSURE_TIME_BLUE_PROP = "exposureTimeBlue"
+  val EXPOSURE_TIME_BLUE_PROP = "blueexposureTime"
```

Comment:
Should be consistent with capitalization of `redExposureTime`.

Reply:
Thanks for catching that.